.github/workflows/publish.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    name: Publish on crates.io
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rs/toolchain@v1
```

Comment:
`action-rs` is archive and unmaintained. There is no widely used and maintained github action for rust. The best practice is to manually install rustup and run build your crate.

Reply:
actually everything seems included by default now (I suppose that’s why the cation deprecated)